### DISCUSSION TITLE
Bollo virtuale automatico e opzione "a carico del cliente"

### DISCUSSION THREAD

**Discussion by Picaland** [fattureincloud/api#165] [Q&A] [answered]
Buongiorno a tutti,
Volevo sapere se durante la creazione della fattura  usando il campo "stamp_duty" 
https://developers.fattureincloud.it/api-reference/#post-/c/-company_id-/issued_documents
presente nello schema CreateIssuedDocumentRequest

se impostato con un valore diverso da 0 (zero) viene valorizzato in automato il campo con il valore e spunta la checkbox "Includi marca da bollo a tuo carico" ?

<img width="469" alt="Schermata 2022-11-18 alle 10 50 56" src="https://user-images.githubusercontent.com/5677228/202673101-9cc89f00-d184-426d-8748-3752b69c5c83.png">

Inoltre volevo capire se esiste un parametro da usare con CreateIssuedDocumentRequest che mi permetta in modo opzionale di impostare il bollo virtuale a carico del cliente.

Grazie!

**Comment by emajo** [comment, marked as answer]
Ciao @Picaland si, il campo `stamp_duty` in un documento elettronico si riferisce alla checkbox `Includi marca da bollo a tuo carico`, invece per avere la marca da bollo a carico del cliente sarà necessario aggiungere un nuovo item all'interno del campo `items_list` con nome "Bollo in fattura", `net_price = 2` e `vat.id=21`.

Grazie,
Emanuele.
